docs/getting_started/user-guide.rst

```diff
@@ -1228,6 +1260,63 @@ it could be that a different method of choosing or sorting tags (beyond the defa
 and filter) is needed.
 
 
+Sync Registry
+-------------
+
+A sync is when we take your local filesystem registry, and retrieve updates from the remote defined at 
+``sync_registry`` in your settings.yaml. Since sync will be writing recipes to the filesystem
+it only works if you target a filesystem registry (meaning that the default registry
+as a remote will not work). 
+
+.. note::
+
+   By default, the first filesystem registry found in your settings under the registry list will
```

Comment:
Hmm, I like this idea, but I don't think it belongs in the settings.yaml - the reason because there is a distinction between defining your registries and then asking for sync. What about a yaml file that you provide to sync instead?

Reply:
If we're talking about adding a yaml file, what about defining a standard one to put at the root of the local registry, and that sync would automatically use ? Similar in a way to view.yaml at the root of each view.

The file could initially just contain the URL of the remote, but could be expanded. First thing that comes to my mind would be to allow the local registry to be in sync with a sub-directory of the remote, e.g.
```
url: https://github.com/remote
path: my_registry
```
the remote git repository would have a directory named `my_registry` at the root, and only it would be synchronised.

Reply:
@muffato I don't think we'd need this by default - the main shpc can work entirely without any registry (just the remote in the config) and certainly a remote doesn't need the reference. I think we should provide examples that people can copy, but it shouldn't be some default or required thing. To summarize the use cases:

1. I'm using shpc installing from a remote registry, and I don't care to add or sync. The default works without extra files. (basic)
2. I'm using shpc install from a remote and wanting to sync recipes locally to tweak. I can just add the local registry to my settings.yaml (intermediate)
3. I have several remote registries I want to always sync from, and I'll add an extra file to specify this design (advanced).

Others can chime in, but I think 1 and 2 are the common use cases, and 3 is less common, and given this, I don't want to have it be a requirement for a registry, but rather something that is easy to add if you want that advanced functionality. And it would look like:

```
registry: ["/path/to/local1", "https://remote1", "/path/to/local2", "/path/to/local3"]
sync_registry:
    "/path/to/local1": "https://sync_remote1"
    "/path/to/local3": "https://sync_remote3"
```
as you specified.  Thoughts?

Reply:
okay this is added (pending passing tests). The first bit seems redundant if we just need the key value pairs, so I only added `sync_registry`